lib/services/access_list.go

```diff
@@ -210,11 +211,28 @@ func IsAccessListOwner(identity tlsca.Identity, accessList *accesslist.AccessLis
 	return nil
 }
 
+// AccessListMemberAndLocker is an interface that allows getting access list members and detecting if they're locked.
+type AccessListMemberAndLockGetter interface {
+	AccessListMembersGetter
+	LockGetter
+}
+
 // IsAccessListMember will return true if the user is a member for the current list.
-func IsAccessListMember(ctx context.Context, identity tlsca.Identity, clock clockwork.Clock, accessList *accesslist.AccessList, memberGetter AccessListMembersGetter) error {
+func IsAccessListMember(ctx context.Context, identity tlsca.Identity, clock clockwork.Clock, accessList *accesslist.AccessList, getter AccessListMemberAndLockGetter) error {
```

Comment:
There's getting to be quite a lot of arguments here. Might be time to consider making this a method on a struct, and the struct can hold the clock and the two clients..